en/developer-docs/docs/administer/configure-a-custom-domain-for-your-organization.md

```diff
@@ -6,7 +6,7 @@ This section provides an overview of Choreo’s custom domain configuration mode
 
 ## Choreo custom domain configuration model
 
-Choreo allows organization administrators to add custom domains for their organizations. When an administrator adds custom domains to an organization, component developers can submit requests to utilize the custom domains for their respective components. These requests require approval from the organization administrator. Upon approval, the custom domain and the relevant URL customization become available to the component.
+Choreo allows organization administrators to add custom domains for their organizations. When an administrator adds custom domains to an organization, component developers can utilize the custom domains for their respective components. 
```

Comment:
```suggestion
Choreo allows organization administrators to add custom domains for their organizations. When an administrator adds custom domains to an organization, developers can utilize the custom domains for their respective components. 
```

IMO it's better to drop the `component` part. WDYT?

Reply:
yes better to remove it. Will make the change

Reply:
addressed with fa21322624c9540ef207ec3ef0ef7a59a2024e9b